doc/connectivity/bluetooth/api/mesh/rpr_srv.rst

```diff
@@ -15,8 +15,7 @@ The Remote Provisioning Server does not have an API of its own, but relies on a
 :ref:`bluetooth_mesh_models_rpr_cli` to control it. The Remote Provisioning Server
 model only accepts messages encrypted with the node's device key.
 
-If present, the Remote Provisioning Server model must be instantiated on the primary
-element.
+If present, the Remote Provisioning Server model must only be instantiated on the primary element.
```

Comment:
This is incorrect. The specs says: "If supported, the Remote Provisioning Server shall be supported by a primary element **and may be supported** by any secondary elements."

This should perhaps also be specified in the client model.